### ISSUE TITLE
Migrate to Peril Staging

### ISSUE THREAD

**Issue by ashfurrow** [ashfurrow/peril-settings#9]
Basically just https://github.com/Moya/moya-peril/issues/30 but for my own settings.

**Comment by ashfurrow** [comment]
Ran into a problem after selecting my org. The screen was white, no errors in the console. I went to the main dashboard URL and the page was rendered with formatting, with "ashfurrow" as an org that needed setup:

![screen shot 2018-09-09 at 4 21 55 pm](https://user-images.githubusercontent.com/498212/45268593-7e0b8b00-b44c-11e8-8af5-44f96fd59b00.png)

When I clicked the link, it produced the following error:

![screen shot 2018-09-09 at 4 19 38 pm](https://user-images.githubusercontent.com/498212/45268569-40a6fd80-b44c-11e8-9c97-7f530ea69620.png)

Refreshing the page subsequently caused a 500. It has a setup ID in the URL I can provide /cc @orta.

**Comment by orta** [comment]
ah, looks like I never made the decision on `staging-dashboard.peril.systems` vs `staging.peril.systems/dashboard`

**Comment by orta** [comment]
Hrm, I can't find how you even ended up on `staging.` domain

<img width="489" alt="screen shot 2018-09-09 at 8 56 31 pm" src="https://user-images.githubusercontent.com/49038/45271169-eff6cb00-b472-11e8-84e9-d7130358dd8a.png">

**Comment by ashfurrow** [comment]
I clicked the link from the [homepage](https://staging.peril.systems), maybe that was it? `https://staging-api.peril.systems/api/auth/peril/github/start?redirect=https%3A%2F%2Fstaging.peril.systems%2Fdashboard`

**Comment by ashfurrow** [comment]
Got this set up 🎉 I ran into problems where Peril Staging should have commented on [this issue](https://github.com/ashfurrow/test-repo/issues/15) but did not. I tried to look at the logs, but the websocket url (`wss://staging-api.peril.systems/primus?iID=327444&_primuscb=...'`) 404'd. I would remove it and try adding the integration again, but it doesn't look possible through the web interface yet.

I'll try taking a look with Orta this week/end.
